future/src/ct/ct_storage_control.cc

```diff
@@ -28,7 +28,17 @@
 #include <glib/gstdio.h>
 
 
-static CtStorageEntity* get_entity_by_type(CtMainWin* pCtMainWin, CtDocType file_type)
+Glib::ustring CtStorageControl::_check_and_unpack(const Glib::ustring& path) {
```

Comment:
I know it's not consistent at the moment but all future code (and to be gradually fixed in existing code) std::string is to be used for filepaths, not Glib::ustring